### DISCUSSION TITLE
Failed patterns 8/9

### DISCUSSION THREAD

**Discussion by jackpot0000000000001** [dragonyue0417/SmartHunter#5] [Q&A]
Every time I try to use Smart Hunter there is always patterns failed 8/9 is there a fix for this I added a screen capture 
![Capture](https://user-images.githubusercontent.com/87583675/126048244-136b80f5-efa0-4222-867c-e8c689d3b154.JPG)

**Comment by dragonyue0417** [comment]
Run the game first, then start smarthunter.
